apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableSettingsPartial.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright 2018 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using Google.Api.Gax.Grpc;
+using Grpc.Core;
+using System;
+
+namespace Google.Cloud.Bigtable.V2
+{
+    public partial class BigtableSettings
+    {
+        /// <summary>
+        /// <see cref="CallSettings"/> for synchronous and asynchronous calls to
+        /// <c>BigtableClient.MutateRow</c> and <c>BigtableClient.MutateRowAsync</c>
+        /// when the request is known to be idempotent.
+        /// </summary>
+        /// <remarks>
+        /// The default <c>BigtableClient.MutateRow</c> and
+        /// <c>BigtableClient.MutateRowAsync</c> <see cref="RetrySettings"/> for idempotent requests are:
+        /// <list type="bullet">
+        /// <item><description>Initial retry delay: 100 milliseconds</description></item>
+        /// <item><description>Retry delay multiplier: 1.3</description></item>
+        /// <item><description>Retry maximum delay: 60000 milliseconds</description></item>
+        /// <item><description>Initial timeout: 20000 milliseconds</description></item>
+        /// <item><description>Timeout multiplier: 1.0</description></item>
+        /// <item><description>Timeout maximum delay: 20000 milliseconds</description></item>
+        /// </list>
+        /// Retry will be attempted on the following response status codes:
+        /// <list>
+        /// <item><description><see cref="StatusCode.DeadlineExceeded"/></description></item>
+        /// <item><description><see cref="StatusCode.Unavailable"/></description></item>
+        /// </list>
+        /// Default RPC expiration is 600000 milliseconds.
+        /// </remarks>
+        public CallSettings IdempotentMutateRowSettings { get; set; } = CallSettings.FromCallTiming(
```

Comment:
I suspect this doesn't want to be a `CallSettings`, but just a `RetrySettings`, given where it's being applied.

Reply:
Done